src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -2398,3 +2398,56 @@
 type: group
 short-summary: Manage log analytics workspace for a vm.
 """
+
+helps['vm monitor metrics'] = """
+type: group
+short-summary: Manage metrics for a vm.
+"""
+
+helps['vm monitor metrics tail'] = """
+type: command
+short-summary: List the metric values for a vm.
+parameters:
+  - name: --aggregation
+    short-summary: The list of aggregation types (space-separated) to retrieve.
+    populator-commands:
```

Comment:
Do you know where these keys (e.g. populator-commands) come from?

Reply:
well, actually, I don't know. I copy these help from monitor part since it's two separate commands.

Reply:
Curious about what additional 'populator-commands' does here